src/auth.js

```diff
@@ -225,6 +226,31 @@ function init(app, db, domain) {
             }
         );
     }
+
+    // Amazon Cognito oAuth
+    if (authConfig.cognito) {
```

Comment:
This will always pass as the config exists, but the values may not be set. Could you confirm what happens if it gets into this if, but doesn't work?

Reply:
Thanks, @DesignyourCode. The convention in the rest of the system is to simply delete the setting in `config.json` if it doesn't exist (if you have a look at the `config.tpl.json` file, you'll see what I'm on about).

If the setting isn't correct in the config file, when the user attempts to auth, they'll receive an error from Amazon Cognito (invalid Endpoint, or some-such).

Reply:
I'll add some additional checks to the _Cognito_ checks, in `auth.js`.